modules/dataquery/jsx/definefilters.addfiltermodal.tsx

```diff
@@ -415,14 +431,14 @@ function valueInput(fielddict: FieldDictionary,
   case 'boolean':
     return <FilterableSelectGroup groups={
       {'Value': {
-        'true': 'true',
-        'false': ' false',
+        'true': t('true', {ns: 'dataquery'}),
```

Comment:
"True" and "False" already exist in the loris namespace. Can you just use those? (or call toLowerCase on the result if necessary)?